### ISSUE TITLE
Question about optimal hyper parameters for training Qwen-Image-Edit-2509

### ISSUE THREAD

**Issue by dylanzonix** [modelscope/DiffSynth-Studio#994]
DiffSynth is great. I am trying to train Qwen-Image-Edit-2509. I have a dataset of 5.4k examples. I'm curious if full parameter fine-tuning could be feasible with this dataset size, and if so at what hyperparameters? I've tried 1e-5 and 5e-6 at a 128 effective batch size and the model degrades when I evaluate it.

Base model:
<img width="1024" height="1024" alt="Image" src="https://github.com/user-attachments/assets/8e321db8-3899-47a2-88a9-a528908e6a41" />

Step 1000 after training (clearly degraded in quality):
<img width="1024" height="1024" alt="Image" src="https://github.com/user-attachments/assets/a6bb042c-5439-4fa9-a467-90ec1a22cb07" />

The loss barely budges:
<img width="2528" height="1328" alt="Image" src="https://github.com/user-attachments/assets/1653e7ce-16ce-47b6-b616-d7cdb4e0e9ce" />

The next thing I'll try is LoRA, but I'm just confused how I couldn't get the training to have any positive impact on the model at any point across a variety of training runs. Curious if anyone, besides the Qwen team, has ever successfully fine-tuned this model. Thanks!

**Comment by Artiprocher** [comment]
@dylanzonix We have also observed similar issues: the Qwen-Image-Edit and Qwen-Image-Edit-2509 models are difficult to fine-tune, despite following exactly the same mathematical principles as Qwen-Image. If we find a better improvement approach, we will update the code accordingly.

**Comment by dylanzonix** [comment]
> [@dylanzonix](https://github.com/dylanzonix) We have also observed similar issues: the Qwen-Image-Edit and Qwen-Image-Edit-2509 models are difficult to fine-tune, despite following exactly the same mathematical principles as Qwen-Image. If we find a better improvement approach, we will update the code accordingly.

Thanks a lot. I am still actively working on this as well and can give updates from my end as well. I'm also happy to share with you the dataset I am using if you are interested. It's custom made exactly for Qwen-Image-Edit-2509

**Comment by yangguoquan001** [comment]
I also encounter the same problem: the loss intend to oscillation instead of converging when fine tuning qwen-image-edit and qwen-image-edit-2509 with lora.

**Comment by clytze0216** [comment]
same question , is there any solutions?

**Comment by lanson07** [comment]
same question

**Comment by iva-jhsun** [comment]
同样的问题

**Comment by DreamOneYou** [comment]
Training the LoRa model for Qwen Image Edit 2509 using ModelScope significantly reduced the loss. However, training with DiffSyhtn resulted in a persistently oscillating loss.

**Comment by dylanzonix** [comment]
> Training the LoRa model for Qwen Image Edit 2509 using ModelScope significantly reduced the loss. However, training with DiffSyhtn resulted in a persistently oscillating loss.

Interesting. You think the issue could be with the DiffSynth library's implementation?

**Comment by Deng-Xian-Sheng** [comment]
> > Training the LoRa model for Qwen Image Edit 2509 using ModelScope significantly reduced the loss. However, training with DiffSyhtn resulted in a persistently oscillating loss.
> 
> Interesting. You think the issue could be with the DiffSynth library's implementation?

Has the problem been solved?

**Comment by 116316018** [comment]
Could you please tell me how to add a loss log?

**Comment by Linshou99** [comment]
> Training the LoRa model for Qwen Image Edit 2509 using ModelScope significantly reduced the loss. However, training with DiffSyhtn resulted in a persistently oscillating loss.

Hi, would you mind sharing which ModelScope framework you used for training Qwen Image Edit 2509?

**Comment by minushuang** [comment]
hello，is there any solutions?
